internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -96,6 +98,30 @@ func (r *ClusterDeploymentsReconciler) Reconcile(ctx context.Context, req ctrl.R
 		return ctrl.Result{}, nil
 	}
 
+	if clusterDeployment.ObjectMeta.DeletionTimestamp.IsZero() { // clusterDeployment no being deleted
```

Comment:
Did you test it with Hive? trying to understand if there will be a collision with Hive

Reply:
Good point. I'll look into it.